tests/subsys/llext/simple/testcase.yaml

```diff
@@ -36,50 +36,6 @@ tests:
     extra_configs:
       - arch:arm:CONFIG_ARM_MPU=n
       - CONFIG_LLEXT_STORAGE_WRITABLE=y
-  llext.simple.modules_enabled_writable:
-    filter: not CONFIG_MPU and not CONFIG_MMU
-    platform_key:
-      - simulation
-      - arch
-    platform_exclude:
-      - qemu_cortex_a9 # MMU
-    extra_configs:
-      - arch:arm:CONFIG_ARM_MPU=n
-      - CONFIG_MODULES=y
-      - CONFIG_LLEXT_STORAGE_WRITABLE=y
-      - CONFIG_LLEXT_TEST_HELLO=m
-  llext.simple.modules_enabled_writable_relocatable:
-    arch_exclude: arm arm64
-    filter: not CONFIG_MPU and not CONFIG_MMU
-    integration_platforms:
-      - qemu_xtensa
-    extra_configs:
-      - CONFIG_MODULES=y
-      - CONFIG_LLEXT_STORAGE_WRITABLE=y
-      - CONFIG_LLEXT_TYPE_ELF_RELOCATABLE=y
```

Comment:
as an example, I looked for another `CONFIG_LLEXT_STORAGE_WRITABLE=y && CONFIG_LLEXT_TYPE_ELF_RELOCATABLE=y` test here since you're saying, that you're removing copies, and I haven't found such a test, am I missing something?

Reply:
No, you're right - that was a proper mistake. :slightly_smiling_face: 
I will make sure to check all removed paths and convert them to non-MODULES in case they were not duplicates.